www/docs/mp/index.php

```diff
@@ -245,7 +245,11 @@
 if (count($current_offices) > 0) {
     $position = $current_offices[0];
 } elseif (count($former_offices) > 0) {
-    $position = $former_offices[0];
+    # only use the position if it's current, i.e.
+    # it does not have an end date
+    if (!$former_offices[0]->to_date) {
```

Comment:
Yes, you're right. No idea what I was thinking there :/

I think probably not at all is my preference. Mostly as for e.g Jack Straw this would list him as "Former Chair, House of Commons Governance Committee" which is probably the least relevant position we could list :)